cfn-resources/project/cmd/resource/resource.go

```diff
@@ -61,63 +61,72 @@ func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler
 	if currentModel.Profile == nil || *currentModel.Profile == "" {
```

Comment:
not for this PR but we might want to extract common logic like this default profile one as we're doing the same in all resources

Reply:
agree, this is copied in all resources